container/network.go

```diff
@@ -8,6 +8,15 @@ import (
 	docker "github.com/docker/docker/client"
 )
 
+// NetworkEventType is a type to define the kind of event related to the network
+type NetworkEventType string
+
+// List of Network status to listen to confirm a network deletion
+const (
+	NetworkRemove  NetworkEventType = "remove"
+	NetworkDestroy NetworkEventType = "destroy"
```

Comment:
Small change, but in go the common rule is to put a type of variable only in the first line.

Also I'd remove `Network` prefix and stick to `type EventType string`, following:

```
const (
  EventRemove EventType = "remove"
  // ...
)
```

Check out https://github.com/moby/moby/blob/master/api/swagger.yaml#L7138 -> the name of events are the same for different docker objects

Reply:
Good point, fixing this